core/lib/src/server.rs

```diff
@@ -396,7 +407,8 @@ impl Rocket {
         self.fairings.pretty_print_counts();
         self.fairings.handle_launch(&self);
 
-        launch_info!("{}{} {}{}",
+        trace::info!(target: "launch",
```

Comment:
This one is `launch` but the config one is `rocket::support`. Is this intended? Why are these different?